systemtest/src/test/java/io/strimzi/systemtest/rollingupdate/RollingUpdateST.java

```diff
@@ -341,7 +344,7 @@ void testKafkaAndZookeeperScaleUpScaleDown(ExtensionContext extensionContext) {
         assertThat(kubeClient().listPersistentVolumeClaims(testStorage.getNamespaceName(), testStorage.getClusterName()).stream()
```

Comment:
This assertion will fail in KRaft mode but pass in ZK mode. I need help here. 
In ZK mode, the pvc got deleted soon after Kafka pod is deleted. But in KRaft mode, the pvc will stay for ~2 mins after kafka pod deleted, and then deleted by itself. Do you know why this happened?
Should I look into it? Or workaround it by adding a helper waitPVCDeleted method to fix it? 
I'd like to ask for your advice first.

Thanks.

Reply:
I think it is just a race condition when PVC waits to be garbage collected, `waitPVCDeleted` is a good idea from my POV. @scholzj is my assumption correct?